src/routers/productExports.ts

```diff
@@ -33,9 +36,7 @@ export interface ProdExportReq {
  */
 export interface ProdExportUpdateReq {
   uuid: string;
-
   transportId?: string;
-  datetime?: string;
```

Comment:
i thought we will let them change datetime? and shouldnt we also let them change the recorderuuid too?

Reply:
I forgot :P